src/cogent3/core/new_sequence.py

```diff
@@ -1700,10 +1702,12 @@ def trim_stop_codon(
         If sequence contains gap characters, the result preserves the sequence
         length by adding gap characters at the end.
         """
+        from cogent3.core import new_genetic_code
```

Comment:
Is this ok? Problems with circular imports

Reply:
This is fine, leaving it as import module name (not from module name import)